source/cellular_bg96_api.c

```diff
@@ -2828,6 +2967,8 @@ CellularError_t Cellular_SocketConnect( CellularHandle_t cellularHandle,
     {
         cellularStatus = storeAccessModeAndAddress( pContext, socketHandle, dataAccessMode, pRemoteSocketAddress );
     }
+    LogError( ( "Cellular_SocketConnect: socket Handle %d, %d, %s, %d, %d", socketHandle->contextId, socketHandle->socketId, 
+            socketHandle->remoteSocketAddress.ipAddress.ipAddress, socketHandle->remoteSocketAddress.port, socketHandle->socketState ) );

```

Comment:
Revert this log message, or set it to LogDebug

Reply:
Done